Pipfile

```diff
@@ -44,6 +44,9 @@ kafka-python = ">=2.0.1"
 cachetools = ">=4.1.0"
 pyarrow = ">=0.17.1"
 kombu = ">=4.6.10"
+zipp = "*"
+typed-ast = "*"
+importlib-metadata = "*"
```

Comment:
Why are these now required?

Reply:
Builds were failing on Fedora without importlib-metadata.
The other two, I thought were already required. I can take them out and re-verify.

Reply:
@maskarb 
I've taken out zipp, typed-ast.